js/common/lib/tensor.ts

```diff
@@ -234,5 +234,71 @@ export interface TensorConstructor {
   // #endregion
 }
 
+export interface ImageFromTensorOptions {
+  format?: 'RGB'|'RGBA'|'BGR'|'RBG';
```

Comment:
above each item(property in the interface), use the [JSDoc](https://jsdoc.app/about-getting-started.html) comments to describe each item.

Those comments will be parsed by toolsets to generate API documents.

for example
```ts
  /**
   * Specify the image format of the tensor to be constructed. Assume 'RGB' if ommitted.
   */
  format?: ImageFormat;
```

Reply:
Good point - I updated the code. testing and pushing it shortly